jicofo/src/test/kotlin/org/jitsi/jicofo/CodecConfigTest.kt

```diff
@@ -25,6 +25,11 @@ import org.jitsi.jicofo.codec.Config.Companion.config
 class CodecConfigTest : ShouldSpec() {
     init {
         context("Default configuration") {
+            config.av1.enabled() shouldBe false
+            config.av1.pt() shouldBe 41
```

Comment:
With the way the config system is written you're not allowed to use `pt()` unless the codec is enabled. Also, when av1 is disabled `av1.rtx.enabled()` will return false. I think you can just remove those 3 lines from the test